kong-3.4.0-0.rockspec

```diff
@@ -253,6 +253,7 @@ build = {
     ["kong.db.migrations.core.017_300_to_310"] = "kong/db/migrations/core/017_300_to_310.lua",
     ["kong.db.migrations.core.018_310_to_320"] = "kong/db/migrations/core/018_310_to_320.lua",
     ["kong.db.migrations.core.019_320_to_330"] = "kong/db/migrations/core/019_320_to_330.lua",
+    ["kong.db.migrations.core.020_330_to_340_wasm"] = "kong/db/migrations/core/020_330_to_340_wasm.lua",
```

Comment:
reviewers: ignore this for now. This change is here because our rockspec linter will complain and cause a test failure otherwise. The migration file needs to be renamed before we can merge to master anyways.